lms/services/canvas_studio.py

```diff
@@ -516,6 +519,10 @@ def _admin_email(self) -> str:
             "canvas_studio", "admin_email"
         )
         if not admin_email:
+            LOG.debug(
```

Comment:
Do you have a quick way of replicating this locally? 

In theory this line here:

https://github.com/hypothesis/lms/blob/main/lms/views/api/exceptions.py#L212

Would create an event row based on the launch information if that's available. That could contain  AI, user, course... 

Give it a try locally and maybe we don't need this debug line (although more logging is always welcomed) and we can replace a papertrail search but a more structured metabase query (or metabase slack alert like we do for other types of errors).